public/dist/js/restaurant-week.app.js

```diff
@@ -14,13 +14,12 @@
 	var constants = {
 		urls: {
 			templates: {
-				restaurant: '//staging.baltimorecountymd.gov/_Restaurant Week/app/restaurant.template.html',
-				restaurantList: '//staging.baltimorecountymd.gov/_Restaurant Week/app/restaurant-list.template.html'
+				restaurant: '/_TEST/_templates/restaurant/restaurant.template.html',
+				restaurantList: '/_TEST/_templates/restaurant-list/restaurant-list.template.html'
```

Comment:
I take it this path change came over from a previous pull request?

Reply:
actually this isn't right, I need to rebuild